src/main/webapp/gameFunctions.js

```diff
@@ -5,3 +5,39 @@ function getUserQuizStatus() {
         }
     });
 }
+
+/** 
+* fetches users of the game from the server 
+* and adds score, id and name to the page
+*/
+function loadUsers() {
+    fetch('/populate-leaderboard').then(response => response.json()).then((users) => {
+        const userListElement = document.getElementById('leader-board');
+        for(i = 0; i < users.length; i++){
+            userListElement.appendChild(createUserElement(users[i]));
```

Comment:
since you already have to loop through all the users, can we compare their scores too so that we can put them on the leaderboard in order?

Reply:
It's put in order of score by the servlet